backend/aopp.go

```diff
@@ -179,6 +179,15 @@ func (backend *Backend) aoppKeystoreRegistered() {
 	var accounts []account
 	var filteredDueToScriptType bool
 	for _, acct := range backend.accounts {
+		var accountFingerprint, err = acct.Config().Config.SigningConfigurations.RootFingerprint()
```

Comment:
strange syntax, why not the usual `accountFingerprint, err := ...`?

Reply:
lol, copy+paste went bad :see_no_evil: